src/helpers/figures/charts/chart_common.ts

```diff
@@ -111,6 +127,18 @@ export function copyLabelRangeWithNewSheetId(
   return range ? copyRangeWithNewSheetId(sheetIdFrom, sheetIdTo, range) : undefined;
 }
 
+/**
+ * Copy a title reference. If the title is a reference to a range on the
+ * sheetIdFrom, the range will target sheetIdTo.
+ */
+export function copyTitleReferenceWithNewSheetId(
+  sheetIdFrom: UID,
+  sheetIdTo: UID,
+  title: Range | undefined
+): Range | undefined {
+  return title ? copyRangeWithNewSheetId(sheetIdFrom, sheetIdTo, title) : undefined;
+}
```

Comment:
The function should probably take directly the `ChartTitle` as argument. That way we wouldn't need to do a ternary each time we want to call it